### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mattyanselmo**

### CODE REVIEW

File: config/tables.R [hunk 1]
```diff
@@ -431,6 +431,65 @@ tables_config <- list(
     Shooting_count_actions = list(
         app_name = "Shooting Actions"
     ),
+
```
Comment (bgrnwd) [inline config/tables.R:434]: Not a huge issue, but some of the spacing here is weird. Maybe we should look into a linter for R

File: utils/tables/ui_utils.R [hunk 2]
```diff
@@ -329,7 +329,7 @@ tables_controlbar <- function(page, league_config, tables_rv) {
                 column(12,
                        h4("Player Settings"),
                        tables_cb_numeric(page, league_config, tables_rv,
-                                         "Minimum Minutes Played", "minimum_minutes", 25),
+                                         "Minimum Minutes Played", "minimum_minutes", 250),
```
Comment (bgrnwd) [inline utils/tables/ui_utils.R:332]: What does the `minimum_minutes` here control? What players appear on the web app? Are you changing it for players besides goalkeepers?

**Review by bgrnwd — APPROVED** [review]

File: utils/tables/ui_utils.R [hunk 3]
```diff
@@ -210,7 +210,7 @@ tables_controlbar <- function(page, league_config, tables_rv) {
...
```
Comment (tyler-richardett) [inline utils/tables/ui_utils.R:213]: Can we revert all these back to 25? 250 minutes is too big a jump when using the keyboard keys to slide these values up and down.

File: config/tables.R (hunk 1 above)
Comment (tyler-richardett) [reply config/tables.R:434]: For this config file specifically, we should probably instead convert it to YAML or something a little more human-readable.

**Review by tyler-richardett — CHANGES_REQUESTED** [review]

File: config/tables.R (hunk 1 above)
Comment (bgrnwd) [reply config/tables.R:434]: I wouldn't mind YAML or TOML

**Comment by mattyanselmo** [comment]
I did it. I merged. Felt like a real boss using admin privileges.

**Pull request merged**